### PR TITLE
Offline Storage for Progressive Web Apps. Should fix #4001

### LINKED ISSUE #4001: Updates for "Offline Storage for Progressive Web Apps" article
Article: https://developers.google.com/web/fundamentals/instant-and-offline/web-storage/offline-for-pwa

The storage limits for Edge are missing, available here: https://developer.microsoft.com/en-us/microsoft-edge/platform/documentation/dev-guide/storage/IndexedDB/

Additionally, there is a typo: "Firebox" should be "Firefox".

### PR DESCRIPTION
**Pull request by garanj**

### CODE REVIEW

File: src/content/en/fundamentals/instant-and-offline/web-storage/offline-for-pwa.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ sufficiently mature or standardized to encourage widespread adoption yet.
       <td>IE10</td>
       <td>&lt;250MB</td>
     </tr>
+    <tr>
+      <td>Edge</td>
+      <td><a href="https://developer.microsoft.com/en-us/microsoft-edge/platform/documentation/dev-guide/storage/IndexedDB/">Dependent on volume size</a>
```
Comment (PaulKinlan) [inline src/content/en/fundamentals/instant-and-offline/web-storage/offline-for-pwa.md]: Needs a closing `</td>`

**Comment by PaulKinlan** [comment]
Looks great (thanks), but it needs one fix

**Comment by garanj** [comment]
Ah sorry Paul, that was pretty poor. Have added another commit. On the upside, I'm learning more about git :-)

**Comment by petele** [comment]
@PaulKinlan - can you review and approve your requested changes. Feel free to merge if you're happy.

**Comment by PaulKinlan** [comment]
Looks great!

**Pull request merged**